### PR TITLE
Models refactor

### PR DESCRIPTION
**Pull request by IRostom**

### CODE REVIEW

**Comment by IRostom** [comment]
@MahmoudAziz1996 this is ready for review and merging if there is no problem

File: app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt [hunk 1]
```diff
@@ -91,19 +93,21 @@ object XpayUtils {
...
+        val bodyPay = PayRequestBody()
+
+        variableAmountID?.let { bodyPay.variable_amount_id = it }
+        communityId?.let { bodyPay.community_id = it }
 
         // Payment method
         payUsing?.let {
             if (it in activePaymentMethods) {
-                body?.pay_using = it
+                bodyPay.pay_using = it
```
Comment (MahmoudAziz1996) [inline app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt:104]: pay_using here must be string 
 requestBody["pay_using"] = it.toString().toLowerCase(Locale.ROOT)

Comment (MahmoudAziz1996) [inline app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt]: requestBody["pay_using"] = it.toString().toLowerCase(Locale.ROOT)

**Review by MahmoudAziz1996 — APPROVED** [review]

File: app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt (hunk 1 above)
Comment (MahmoudAziz1996) [reply app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt]: @IRostom

Comment (IRostom) [reply app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt]: done, check now

**Review by MahmoudAziz1996 — APPROVED** [review]

**Pull request merged**